gen/app-gen

```diff
@@ -208,9 +208,18 @@ if [ -z "${manifests_dir}" ]; then
     show_help_and_exit_error
 fi
 
```

Comment:
this modification is only valid for `"docker-compose"` orchestrator and so you have to check if `"$orchestrator" == "docker-compose"`